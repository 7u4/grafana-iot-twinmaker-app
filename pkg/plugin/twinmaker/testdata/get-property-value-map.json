{
    "PropertyValues": {
        "documents": {
            "PropertyReference": {
                "ComponentName": "DocumentComponent",
                "EntityId": "b2f31ce2-0c71-4e6d-8c65-000d8781d676",
                "ExternalIdProperty": null,
                "PropertyName": "documents"
            },
            "PropertyValue": {
                "BooleanValue": null,
                "DoubleValue": null,
                "Expression": null,
                "IntegerValue": null,
                "ListValue": null,
                "LongValue": null,
                "MapValue": {
                    "amazon": {
                        "BooleanValue": null,
                        "DoubleValue": null,
                        "Expression": null,
                        "IntegerValue": null,
                        "ListValue": null,
                        "LongValue": null,
                        "MapValue": null,
                        "RelationshipValue": null,
                        "StringValue": "https://www.amazon.com"
                    },
                    "google": {
                        "BooleanValue": null,
                        "DoubleValue": null,
                        "Expression": null,
                        "IntegerValue": null,
                        "ListValue": null,
                        "LongValue": null,
                        "MapValue": null,
                        "RelationshipValue": null,
                        "StringValue": "https://www.google.com"
                    }
                },
                "RelationshipValue": null,
                "StringValue": null
            }
        }
    }
}