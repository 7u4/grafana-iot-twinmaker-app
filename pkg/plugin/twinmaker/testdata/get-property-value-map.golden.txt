ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {
    "custom": {}
}
Name: documents
Dimensions: 2 Fields by 2 Rows
+-----------------+------------------------+
| Name: Key       | Name: Value            |
| Labels:         | Labels:                |
| Type: []*string | Type: []*string        |
+-----------------+------------------------+
| amazon          | https://www.amazon.com |
| google          | https://www.google.com |
+-----------------+------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////8AEAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAIgAAAADAAAAVAAAACgAAAAEAAAAkP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAACw/v//CAAAABQAAAAJAAAAZG9jdW1lbnRzAAAABAAAAG5hbWUAAAAA2P7//wgAAAAYAAAADQAAAHsiY3VzdG9tIjp7fX0AAAAEAAAAbWV0YQAAAAACAAAA2AAAAAQAAABC////FAAAAKQAAACkAAAAAAAFAaAAAAACAAAALAAAAAQAAAA0////CAAAABAAAAAFAAAAVmFsdWUAAAAEAAAAbmFtZQAAAABY////CAAAAFAAAABHAAAAeyJsaW5rcyI6W3sidGl0bGUiOiJMaW5rIiwidGFyZ2V0QmxhbmsiOnRydWUsInVybCI6IiR7X192YWx1ZS50ZXh0fSJ9XX0ABgAAAGNvbmZpZwAAAAAAAJT///8FAAAAVmFsdWUAEgAYABQAEwASAAwAAAAIAAQAEgAAABQAAABAAAAARAAAAAAABQFAAAAAAQAAAAwAAAAIAAwACAAEAAgAAAAIAAAADAAAAAMAAABLZXkABAAAAG5hbWUAAAAAAAAAAAQABAAEAAAAAwAAAEtleQD/////2AAAABQAAAAAAAAADAAWABQAEwAMAAQADAAAAGAAAAAAAAAAFAAAAAAAAAMDAAoAGAAMAAgABAAKAAAAFAAAAHgAAAACAAAAAAAAAAAAAAAGAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAQAAAAAAAAABAAAAAAAAAAIAAAAAAAAAAAAAAAAAAAACAAAAAAAAAAEAAAAAAAAAAwAAAAAAAAADAAAAAAAAAAAAAAAAIAAAACAAAAAAAAAAAAAAAAAAAAAgAAAAAAAAAAAAAAAAAAAAAAAAAGAAAADAAAAAAAAABhbWF6b25nb29nbGUAAAAAAAAAABYAAAAsAAAAAAAAAGh0dHBzOi8vd3d3LmFtYXpvbi5jb21odHRwczovL3d3dy5nb29nbGUuY29tAAAAABAAAAAMABQAEgAMAAgABAAMAAAAEAAAACwAAAA8AAAAAAADAAEAAAAAAgAAAAAAAOAAAAAAAAAAYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAIgAAAADAAAAVAAAACgAAAAEAAAAkP7//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAACw/v//CAAAABQAAAAJAAAAZG9jdW1lbnRzAAAABAAAAG5hbWUAAAAA2P7//wgAAAAYAAAADQAAAHsiY3VzdG9tIjp7fX0AAAAEAAAAbWV0YQAAAAACAAAA2AAAAAQAAABC////FAAAAKQAAACkAAAAAAAFAaAAAAACAAAALAAAAAQAAAA0////CAAAABAAAAAFAAAAVmFsdWUAAAAEAAAAbmFtZQAAAABY////CAAAAFAAAABHAAAAeyJsaW5rcyI6W3sidGl0bGUiOiJMaW5rIiwidGFyZ2V0QmxhbmsiOnRydWUsInVybCI6IiR7X192YWx1ZS50ZXh0fSJ9XX0ABgAAAGNvbmZpZwAAAAAAAJT///8FAAAAVmFsdWUAEgAYABQAEwASAAwAAAAIAAQAEgAAABQAAABAAAAARAAAAAAABQFAAAAAAQAAAAwAAAAIAAwACAAEAAgAAAAIAAAADAAAAAMAAABLZXkABAAAAG5hbWUAAAAAAAAAAAQABAAEAAAAAwAAAEtleQAgAgAAQVJST1cx
